src/main/java/com/smartcar/sdk/SmartcarVehicleOptions.java

```diff
@@ -41,6 +57,12 @@ private SmartcarVehicleOptions(Builder builder) {
         this.version = builder.version;
         this.unitSystem = builder.unitSystem;
         this.origin = builder.origin;
+        if (builder.flags.size() > 0) {
+            String[] flagStrings = builder.flags.toArray(new String[0]);
+            this.flags = Utils.join(flagStrings, " ");
```

Comment:
```suggestion
            this.flags = String.join(" ", builder.flags);
```

https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.Iterable-